serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -8,6 +8,39 @@ toc::[]
 
 If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest version.
 
+[id="update-api-group_{context}"]
+== Updating the API group
+
+In {ServerlessProductName} 1.4.0 and newer versions, the `KnativeServing` object's API group changed from `serving.knative.dev` to `operator.knative.dev`.
+You must adjust any of your scripts or applications that rely on the old API group to use the new API group.
+
+The {ServerlessOperatorName} creates a `KnativeServing` custom resource (CR) in the new API group for you.
+
+Once you have updated references to the new API group, you must remove any older CR versions and use the newly deployed `KnativeServing` CR instead.
+
+To safely do this without downtime, remove the owner reference from the newly deployed `KnativeServing` CR by entering the following command:
+----
+ $ oc edit knativeserving.operator.knative.dev knative-serving -n knative-serving
+----
+
+After the owner reference has been removed, you can safely remove any older CR versions and start using the new one.
+
+[IMPORTANT]
+====
+If a previous version of the CR exists, changes to the new CR will be overwritten by the {ServerlessOperatorName}. While the old CR is still active, all changes need to be made to that CR.
+====
+
+This change to the API group causes commands without a fully qualified API group and kind, such as `oc get knativeserving`, to become unreliable and not always work correctly.
+
+After upgrading to {ServerlessProductName} 1.6.0, you must remove the old CRD to fix this issue.
+You can remove the old CRD by entering the following command:
+----
+$ oc delete crd knativeservings.serving.knative.dev
+----
+
+[id="upgrade-sub-channel_{context}"]
+== Upgrading the Subscription Channel
+
```

Comment:
Is this heading expected to be here? I don't see any removed lines in this PR so this is new and not just moved.

Reply:
Yeah, it is. It's because we have two sub-sections now since I added **Updating the API group**
It's just to help conform with our modular docs approach and improve readability - make sense?